<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music player</string>
  </property>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QListWidget" name="songsListWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="elapsedTimeSlider">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>20</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="playlistListWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>70</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addPlaylistButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Playlist</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
